locations/spiders/electrify_america.py

```diff
@@ -0,0 +1,31 @@
+import json
+import random
+
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+from locations.items import Feature
+
+
+class ElectrifyAmericaSpider(scrapy.Spider):
+    name = "electrify_america"
+    item_attributes = {"brand": "Electrify America", "brand_wikidata": "Q59773555"}
+    start_urls = ["https://api-prod.electrifyamerica.com/v2/locations"]
+
+    def parse(self, response):
+        for item in response.json():
+            feature = DictParser.parse(item)
+
+            extras = dict()
+            extras["capacity"] = item.get("evseMax")
+
+            if access := item.get("type"):
+                if access == "PUBLIC":
+                    extras["access"] = "public"
+
+            if "extras" not in feature:
+                feature["extras"] = dict()
```

Comment:
We do this alot. Maybe it'd be better to always populate extras as a empty dict?

Reply:
Yea, that probably makes sense.

Reply:
OK, I'll do a PR tomorrow, unless you do it before. I've also been thinking about `opening_hours` we should probably allow that to be `OpeningHours`, and then do `as_opening_hours()` on export, maybe init that too.